### ISSUE TITLE
LaTeX spelling

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#573]
The output strings currently spell it LaTex, but it should be LaTeX if we're following the formal naming convention.

**Comment by GeorgeKerscher** [comment]
The X in TeX is actually the greek Chi, which is set typographically slightlower than the rest of the word. This was done by Donald Knuth to show off the capabilities of Tex for formatting. I don't think we want to put that into our document, so just made it LaTeX! PR coming soon.
